sormas-app/app/src/main/java/de/symeda/sormas/app/core/TimeAgo.java

```diff
@@ -204,41 +225,50 @@ private static long getTimeDistanceInMinutes(long time) {
 	 */
 	private enum Periods {
 
-		NOW(R.string.time_now, new DistancePredicate() {
+		NOW(R.string.time_now, false, new DistancePredicate() {
 
 			@Override
 			public boolean validateDistanceMinutes(final long distance) {
 				return distance == 0;
 			}
 		}),
-		ONEMINUTE_PAST(R.string.time_past_one_minute, new DistancePredicate() {
+		ONEMINUTE_PAST(R.string.time_past_one_minute, false, new DistancePredicate() {
 
 			@Override
 			public boolean validateDistanceMinutes(final long distance) {
 				return distance == 1;
 			}
 		}),
-		XMINUTES_PAST(R.string.time_past_x_minutes, new DistancePredicate() {
+		XMINUTES_PAST(R.string.time_past_x_minutes, false, new DistancePredicate() {
 
 			@Override
 			public boolean validateDistanceMinutes(final long distance) {
 				return distance >= 2 && distance <= 44;
 			}
 		}),
-		ABOUTANHOUR_PAST(R.string.time_past_one_hour, new DistancePredicate() {
+		ABOUTANHOUR_PAST(R.string.time_past_one_hour, false, new DistancePredicate() {
 
 			@Override
 			public boolean validateDistanceMinutes(final long distance) {
 				return distance >= 45 && distance <= 89;
 			}
 		}),
-		XHOURS_PAST(R.string.time_past_x_hours, new DistancePredicate() {
+		XHOURS_PAST(R.string.time_past_x_hours, false, new DistancePredicate() {
 
 			@Override
 			public boolean validateDistanceMinutes(final long distance) {
 				return distance >= 90 && distance <= 1439;
 			}
 		}),
+
+		TODAY(R.string.time_past_today, true, new DistancePredicate() {
+
+			@Override
+			public boolean validateDistanceMinutes(final long distance) {
+				return distance >= 0 && distance <= 1439;
```

Comment:
Add a comment here please that 1440 minutes are 24 hours, and use `< 1440` instead because it's more clear